proxy/src/registry.rs

```diff
@@ -352,6 +360,15 @@ impl Registry {
 
         Ok(tx)
     }
+
+    /// Returns the configured thresholds for [`Transaction`] acceptance stages.
+    #[must_use]
+    pub const fn thresholds() -> Thresholds {
+        Thresholds {
+            confirmation: 3,
+            settlement: 9,
```

Comment:
Not sure these values make sense :thinking: -- `confirmation` should be `1` if it's supposed to represent how many blocks to wait for confirmation.

Reply:
If we're talking about thresholds, we actually only need `settlement`, because `confirmation` is always `1`

Reply:
Isn't confirmation `1` the equivalent of being applied?

Reply:
Yeah -- all I'm saying is we don't need both numbers: when a transaction lands in a block, it can be considered "mined" or "confirmed with deph=1". The number that's configurable is how many blocks (confirmations) to wait for the tx to be settled. This is `6` for bitcoin.

Reply:
It's also confusing because some people use the term "confirmed" to mean "settled".

Reply:
So you could have just `Thresholds { confirmations: 6 }`, and that would do the job. "Wait 6 confirmations before considering the tx settled."